chromium_src/ios/chrome/browser/sync/ios_chrome_sync_client.mm

```diff
@@ -15,7 +15,8 @@
 IOSChromeSyncClient::CreateDataTypeControllers(
     syncer::SyncService* sync_service) {
   syncer::ModelTypeSet disabled_types = {// syncer::PASSWORDS,
-                                         syncer::PROXY_TABS, syncer::AUTOFILL,
+                                         syncer::PROXY_TABS,
+                                         // syncer::AUTOFILL,
```

Comment:
why was autofill removed?

Reply:
When `AUTOFILL` is disabled, password sync was not working properly. But I will make sure this is the case and If it is not needed disable again.

Reply:
Checked and tested, `AUTOFILL` is not needed and disabled again. Thanks for the catch.
Fixed at https://github.com/brave/brave-core/pull/12360/commits/dac3f130d7a958864a32ba18e0ab81c844040ab5